rdr_service/offline/main.py

```diff
@@ -1291,6 +1247,13 @@ def _build_pipeline_app():
         methods=['GET']
     )
 
+    offline_app.add_url_rule(
+        OFFLINE_PREFIX + 'AW3ReadyMissingDataFilesReport',
```

Comment:
Add "Genomic" prefix here as well.

Reply:
I've updated the cron job to use Genomic in the route name